data/json/effects_on_condition/npc_eocs/appearance_eocs.json

```diff
@@ -0,0 +1,325 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_hair_growth_tracking",
+    "//": "Hair usually grows at a rate of 1 cm per month.  This is not a hard number, because hair growth is very complex.",
+    "//2": "For the purposes of this EOC right now, we'll assume this rate.",
+    "//3": "For ease of calculation, all math will be done with centimeters for hair growth.",
+    "//4": "For reference starting point, a buzzcut is 0.32 centimeters long.",
+    "//5": "1 inch is 2.54 centimeters.",
+    "recurrence": [ "75 days", "90 days" ],
```

Comment:
Just a thought: if you want to make hair grow change depending on person's health and/or traits, you can make this recurrence a variable, and change this variable in a separate EoC
So "stop hair grow" can be a trait, that simply zero away the default grow rate

Reply:
I'll probably save that for a follow-up PR, but I'll keep this in mind. So far, I've gotten the health dependency set up.